doc/README.md

```diff
@@ -1,49 +1,157 @@
-Kivy - Documentation
-====================
+Kivy
```

Comment:
This README is a separate one for docs, but now it looks like a copy of the "main" one.

Is that right?

Reply:
Nope. Not right at all.